### ISSUE TITLE
redesign of robot_state_t

### ISSUE THREAD

**Issue by mauricefallon** [openhumanoids/bot_core_lcmtypes#30]
robot_state_t has stood the test of time but it has a couple of issues:
1 it assumes wrist and foot FT sensors.
2 it assumes the foot FT sensors are 3-axis.
3 of less importance is the use of vector3d_t and quaternion_t messages.

MIT have locally modified it to fix 2. 

This should be pushed into master but will mean that all DRC finals logs cannot to replayed.
A simple python script would allow replays of these logs.

on the whole, I'd be happy to do this redesign but am not currently motivated to push it though OH

**Comment by patmarion** [comment]
agreed it might be time to revisit this message, and other messages like command messages.  I'm not actively running atlas or valkyrie, so I don't have a lot of stake in it right now.

regarding DRC logs- perhaps we need to build a VM with a oh-distro snapshot that is capable of playing all the logs out of the box, this will ensure we have a method to replay logs well into the future.  Then we can also try to maintain a set of python translator scripts, but I'm afraid those won't be guaranteed to work forever.

**Comment by RussTedrake** [comment]
FWIW - @tkoolen and I have also been discussing this, and are likely to put an updated version in robotlocomotion/lcmtypes at some point if nobody beats us to it.
